metaflow/plugins/aip/tests/run_integration_tests.py

```diff
@@ -49,13 +49,13 @@
 ]
 
 disabled_test_flows = [
-    "kfp_flow.py",  # kfp_preceding_component feature has been deprecated.
+    "aip_flow.py",  # kfp_preceding_component feature has been deprecated.
     "flow_triggering_flow.py",  # TODO(talebz): will re-enable with Argo FTF
     # TODO(talebz) AIP-6717 re-enable for compilation changes or when cluster can handle
```

Comment:
I see the lines below are commented out, is this work planned ahead of the release?

Reply:
tracking to re-disable nested foreach tests after kfp-argo merge with [AIP-7772](https://zillowgroup.atlassian.net/browse/AIP-7772)